cms/djangoapps/contentstore/views/tests/test_assets.py

```diff
@@ -259,7 +259,7 @@ def test_download(self):
         # Now, download it.
         resp = self.client.get(self.uploaded_url, HTTP_ACCEPT='text/html')
         self.assertEquals(resp.status_code, 200)
-        self.assertEquals(resp.content, self.asset_name)
+        self.assertEquals(''.join(resp), self.asset_name)
```

Comment:
Looks like this was changed to work with `StreamingHttpResponse`, so I won't backport it.